ansible/roles/gcp-cloud-storage/meta/main.yml

```diff
@@ -0,0 +1,2 @@
+dependencies:
```

Comment:
Let's remove the meta directory. The expectation is that a user runs the bootstrap job and installs the required packages first.